TODO.validation-gen

```diff
@@ -61,11 +76,14 @@ P1:
   - uniqueness
   - correlated updates
   - single field manager
+- Correlated updates for listType=set?
+  - sort of meaningless but consistent?
```

Comment:
For CRDs we don't correlate the contents of `listType=set` because it is required to always be atomic.  

For ratcheting I don't think it matters. For a set element, the only reasonable oldObject that could be provided to the validation of a set element would be nil, or the same value as object.. I think?

cc @yongruilin for ratcheting thoughts